grails-app/controllers/org/bbop/apollo/OrganismController.groovy

```diff
@@ -44,26 +44,26 @@ class OrganismController {
     @RestApiParams(params = [
             @RestApiParam(name = "username", type = "email", paramType = RestApiParamType.QUERY)
             , @RestApiParam(name = "password", type = "password", paramType = RestApiParamType.QUERY)
-            , @RestApiParam(name = "organism", type = "string", paramType = RestApiParamType.QUERY, description = "ID or commonName that can be used to uniquely identify an organism")
+            , @RestApiParam(name = "organism", type = "json", paramType = RestApiParamType.QUERY, description = "Pass an Organism JSON object with an 'id' that corresponds to the organism to be removed")
```

Comment:
@deepakunni3  Was this the only actual change (indicating the type properly)?  If so, I think it is good, so long as you re-run `./apollo create-rest-doc` and push that.